--- a/gulpfile.js	2018-01-09 03:05:29.000000000 -0600
+++ b/gulpfile-new.js	2018-01-25 19:59:18.475995880 -0600
@@ -68,7 +68,7 @@
 	const ARCHS={ "64": "x64", "32": "ia32"}
 	var archName = ARCHS[arch];
 	var EXTENSIONS = { "win": ".exe" }
-	var target = "node" + (config.node_major || 7) + "-" +
+	var target = "node" + (config.node_major || 8) + "-" +
 		platformName + "-" + archName;
 	var binaryName = config.id + "-" + platform + 
 		"-" + ARCH_BITS[arch] + (EXTENSIONS[platform] || "");

--- a/config.json	2018-01-09 03:05:29.000000000 -0600
+++ b/config-new.json	2018-01-25 20:21:28.652964007 -0600
@@ -2,7 +2,7 @@
 	"id": "net.downloadhelper.coapp",
 	"name": "VdhCoApp",
 	"description": "Video DownloadHelper companion app",
-	"node_major": 7,
+	"node_major": 8,
 	"extra": [{
 			"source": "converter/build/${platform}/${arch}/"
 		},{
