--- a/converter.js	2018-06-13 07:58:51.000000000 -0600
+++ b/converter-new.js	2018-06-15 01:03:03.404182255 -0600
@@ -44,7 +44,7 @@
 else
 	throw new Error("Unsupported architecture",process.arch);
 
-const binaryDir = path.join(path.dirname(process.execPath),"..","converter","build",platform,arch);
+const binaryDir = platform == "linux" ? "/usr/bin" : path.join(path.dirname(process.execPath),"..","converter","build",platform,arch);
 const binaryPath = path.join(binaryDir,"ffmpeg");
 const probeBinaryPath = path.join(binaryDir,"ffprobe");
 const playBinaryPath = path.join(binaryDir,"ffplay");

--- a/native-autoinstall.js	2018-06-13 07:58:51.000000000 -0600
+++ b/native-autoinstall-new.js	2018-06-15 01:08:45.734576023 -0600
@@ -72,7 +72,10 @@
 		mode = "system";
 	var config;
 	try {
-		config = JSON.parse(fs.readFileSync(path.resolve(path.dirname(process.execPath),"../config.json"),"utf8"));
+        if(process.platform == "linux")
+		    config = JSON.parse(fs.readFileSync(path.resolve(path.dirname(process.execPath),"../share/vdhcoapp/config.json"),"utf8"));
+		else
+            config = JSON.parse(fs.readFileSync(path.resolve(path.dirname(process.execPath),"../config.json"),"utf8"));
 	} catch(err) {
 		DisplayMessage("Cannot read config file: "+err.message,"Error");
 		process.exit(-1);
